spec/engine/miq_ae_method_service/miq_ae_service_model_base_spec.rb

```diff
@@ -104,13 +104,13 @@
     end
 
     it 'loads object from yaml' do
-      expect(YAML.safe_load(svc_service.to_yaml)).to eq(svc_service)
+      expect(YAML.load(svc_service.to_yaml)).to eq(svc_service)
```

Comment:
Suggest we pass `MiqAeMethodService::MiqAeServiceService` to safe_load on this one.

Reply:
I just wanted to see you say Service Service Service 🤣

Reply:
We might need HashWithIndifferentAccess too.  I'll check.

Reply:
Nope, just Service ServiceService was enough